websocket/_app.py

```diff
@@ -1,3 +1,4 @@
+import ssl
```

Comment:
Change this to `from ._ssl_compat import *` like the other files. _ssl_compat.py exists for the edge case where SSL can't be imported